source/Halibut.Tests/UsageFixture.cs

```diff
@@ -86,23 +86,25 @@ public void OctopusCanSendMessagesToPollingTentacle()
         public void OctopusCanSendMessagesToWebSocketPollingTentacle()
         {
             var services = GetDelegateServiceFactory();
+            services.Register<ISupportedServices>(() => new SupportedServices());
             const int octopusPort = 8450;
             AddSslCertToLocalStoreAndRegisterFor("0.0.0.0:" + octopusPort);
 
             try
             {
-                using (var octopus = new HalibutRuntime(services, Certificates.Octopus))
+                using (var octopus = new HalibutRuntime(Certificates.Octopus))
                 using (var tentaclePolling = new HalibutRuntime(services, Certificates.TentaclePolling))
                 {
                     octopus.ListenWebSocket($"https://+:{octopusPort}/Halibut");
                     octopus.Trust(Certificates.TentaclePollingPublicThumbprint);
 
                     tentaclePolling.Poll(new Uri("poll://SQ-TENTAPOLL"), new ServiceEndPoint(new Uri($"wss://localhost:{octopusPort}/Halibut"), Certificates.SslThumbprint));
 
-                    var echo = octopus.CreateClient<IEchoService>("poll://SQ-TENTAPOLL", Certificates.TentaclePollingPublicThumbprint);
-                    for (var i = 0; i < 2000; i++)
+                    var echo = octopus.CreateClient<ISupportedServices>("poll://SQ-TENTAPOLL", Certificates.TentaclePollingPublicThumbprint);
+                    for (var i = 1; i < 100; i++)
                     {
-                        echo.SayHello("Deploy package A" + i).Should().Be("Deploy package A" + i + "...");
+                        var i1 = i;
```

Comment:
Do we need i1? Can't we just use i?

Reply:
It's either that or tell resharper to ignore the capture variable scope warning thing... this seemed easier.